ui/components/institutional/jwt-url-form/jwt-url-form.scss

```diff
@@ -7,6 +7,7 @@
     border-radius: 4px;
     border: 1px solid;
     background-color: var(--color-background-default);
+    color: var(--color-text-default);
```

Comment:
Question: I've noticed the implementation of this component is using native html `textArea`, instead of using the one in component library. would it be helpful to switch to that one and make the styling easier. 
And of course, this doesn't need to be done within the same MR, could be a follow up improvement work. wdyt?

Reply:
Oh amazing, was not aware of it, I've also updated the native html `input` we had in there as well 👍🏼